README.md

```diff
@@ -147,13 +148,25 @@ In a default Archivematica installation, this is `/var/archivematica/sharedDirec
 
 An optional parameter `-u` or `--uuid` may be passed to only reindex the AIP that has the matching UUID.
 
-`--delete` will delete any data found in ElasticSearch with a matching UUID before re-indexing.
+`--delete` will delete any data found in Elasticsearch with a matching UUID before re-indexing.
 This is useful if only some AIPs are missing from the index, since AIPs that already exist will not have their information duplicated.
 
-`--delete-all` will delete the entire AIP ElasticSearch index before starting.
+`--delete-all` will delete the entire AIP Elasticsearch index before starting.
 This is useful if there are AIPs indexed that have been deleted.
 This should not be used if there are AIPs stored that are not locally accessible.
 
+### rebuild-transfer-backlog
+
+*Versions*: Archivematica 1.5, 1.6
```

Comment:
This tools was broken when using AM v1.5 before this change, and it could still be broken. I would be ok with just updating the readme.md to say this tool works only with 1.6